nylas/services/routes.py

```diff
@@ -0,0 +1,76 @@
+import hashlib
+import hmac
+from enum import Enum
+from nylas import APIClient
+
+
+class Routes:
+    def __init__(self, api):
+        """
+        Collection of helpful routes to be implemented in a Python server
+
+        Args:
+            api (APIClient): The configured Nylas API client
+        """
+        self._api = api
```

Comment:
What happens if `api` is `None`? Do we need a check and raise an exception for that scenario?